src/github.com/matrix-org/dendrite/clientapi/routing/room_tagging.go

```diff
@@ -163,20 +163,20 @@ func obtainSavedTags(
 	userID string,
 	roomID string,
 	accountDB *accounts.Database,
-) (string, []gomatrixserverlib.ClientEvent, error) {
+) (string, gomatrixserverlib.RawJSON, error) {
 	localpart, _, err := gomatrixserverlib.SplitID('@', userID)
 	if err != nil {
-		return "", []gomatrixserverlib.ClientEvent{}, err
+		return "",gomatrixserverlib.RawJSON{}, err
 	}
 
 	data, err := accountDB.GetAccountDataByType(
 		req.Context(), localpart, roomID, "tag",
 	)
 	if err != nil {
-		return "", []gomatrixserverlib.ClientEvent{}, err
+		return "", gomatrixserverlib.RawJSON{}, err
 	}
 
-	return localpart, data, nil
+	return localpart, data.Content, nil
```

Comment:
So, `data` here is an array. I cant `.Content` it because of the same. I need a way to get the `Content` of the array and then return it. (I was thinking either we can iterate through the array and obtain all the necessary items, or the JSON conversion -  which is back to the same problem, thus infeasible) 

@anoadragon453  any suggestions?

Reply:
I've found a way to fix this currently, but has introduced a lot of build errors. I'll fix this soon and we can probably move forward :)